lib/osf-components/addon/components/file-browser/template.hbs

```diff
@@ -0,0 +1,167 @@
+<div local-class='OptionBar'>
+    <div local-class='OptionBar__left'>
+        <input
+            data-test-file-search
+            type='text'
+            class='form-control'
+            placeholder='{{t 'general.filter'}}'
+        />
+
+        <ResponsiveDropdown
+            local-class='SortDropdown'
+            @horizontalPosition='left'
+            as |dropdown|
+        >
+            <dropdown.trigger
+                data-test-file-sort-trigger
+                data-analytics-name='Sort files'
+            >
+                <Button>
+                    {{t 'registries.overview.files.sort'}}
+                    <span local-class='SortedBy'>
+                        {{t 'registries.overview.files.sort_by.name_ascending'}}
+                    </span>
+                    <FaIcon @icon='caret-down' />
+                </Button>
+            </dropdown.trigger>
+            <dropdown.content
+                local-class='DropdownList'
+            >
+                <Button @layout='fake-link'>
+                    {{t 'registries.overview.files.sort_by.name_ascending'}}
+                </Button>
+                <Button @layout='fake-link'>
+                    {{t 'registries.overview.files.sort_by.name_descending'}}
+                </Button>
+                <Button @layout='fake-link'>
+                    {{t 'registries.overview.files.sort_by.date_ascending'}}
+                </Button>
+                <Button @layout='fake-link'>
+                    {{t 'registries.overview.files.sort_by.date_descending'}}
+                </Button>
+            </dropdown.content>
+        </ResponsiveDropdown>
+    </div>
+    <div local-class='OptionBar__right'>
+        {{!-- Add storage provider name to data-analytics-name --}}
+        <Button
+            data-test-download-all
+            data-analytics-name='Download all files'
+            @layout='fake-link'
+        >
+            <FaIcon @icon='download' />
+            {{t 'registries.overview.files.download_all'}}
+        </Button>
+        <Button
+            data-test-file-help
+            data-analytics-name='Help'
+            aria-label={{t 'general.help'}}
+            @layout='fake-link'
+        >
+            <FaIcon @icon='question-circle' />
+        </Button>
+    </div>
+</div>
+
+<ul local-class='FileList'>
+    {{#each @manager.displayItems as |item|}}
```

Comment:
This is not for now, but when we get to componentizing the FileList items, I think we'll want to do two: one for folders and one for files. They have enough different functionality that it makes sense for them to be two things.